src/components/Authentication/Authentication.vue

```diff
@@ -9,7 +9,8 @@ v-flex.pa-2(xs-12)
           ref='loginForm'
           :oauth-providers='oauthProviders'
           :forgot-password-url='forgotPasswordUrl'
-          :forgot-password-route='forgotPasswordRoute')
+          :forgot-password-route='forgotPasswordRoute'
+          @forgotpassword='$emit("forgotpassword")')
```

Comment:
@zachmullen Are you OK with bubbling the event up to the parent like this so a consumer could listen for it?

Reply:
Note: App.vue is still passing a raw link.  You can now set routes, but since this app doesn't use a router yet, I figured that was the best option.  It will now correctly route you to the forgot password UI on the girder instance (in the demo)

Reply:
Yep, that's a good practice.